Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -10,6 +10,20 @@ public protocol ComposerVCDelegate: AnyObject {
     func composerDidCreateNewMessage()
 }
 
+/// The possible composer states. An Enum is not used so it does not cause
+/// future breaking changes and is possible to extend with new cases.
+public class ComposerState: RawRepresentable, Equatable {
```

Comment:
I get the idea but I don't understand why it's `class` 🤔

Reply:
It doesn't need to be class yes, forgot to remove it after doing some experiments 😄 Will change it now 👍